department-of-veterans-affairs/vets-website
# Bugfix/vfs pw 9476 recurring events add to cal

## Description
On event detail pages with recurrence, "View other times for this event" panel displays all recurrences' date/time, and an Add to calendar link. The Add to Calendar link on each recurrence points to the Event node page, so is effectively just a page reload. (Recurrences do not each have their own page - you cannot effectively link to a single recurrence.)

The Add to Calendar link at page top will always be date and time of the next upcoming recurrence. So there is currently no way for a user to add a future recurrence to calendar.

## Original issue(s)
department-of-veterans-affairs/va.gov-cms#9476


## Testing done
Local tests:
- Got to `/minneapolis-health-care/events/42754/`
- Slide open the "View other items for this event" content block
- Click an event link
- Observe the ICS file is downloaded
- Add to calendar
- Validate the details are correct.

## Screenshots


## Acceptance criteria
- [x] Add to calendar link on each recurrence should act like the Add to calendar link top of page, and download an ICS file for that recurrence's date/time.

## Definition of done
